.github/workflows/cypress.yml

```diff
@@ -18,15 +18,15 @@ jobs:
       - name: Initialise MAAS
         run: sudo maas init region+rack --maas-url=${{env.MAAS_URL}}/MAAS --database-uri maas-test-db:///
       - name: Install Cypress
-        uses: cypress-io/github-action@master
+        uses: cypress-io/github-action@v2
```

Comment:
This wasn't strictly necessary it just seemed to be what the cypress docs suggested, but can revert this change if there's a reason for it to be on master.